src/styles/clock.scss

```diff
@@ -0,0 +1,46 @@
+.clock {
+  display: block;
+  min-width: $clock-size;
+  min-height: $clock-size;
+  @include centralize;
+  border: 8px solid #fff;
+  background-color: #213672;
+  border-radius: 50%;
+  position: relative;
+
+  &__seconds-arrow,
+  &__minure-arrow {
+    width: 0;
+    position: absolute;
+    animation-name: move-arrow;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+  }
+
+  &__seconds-arrow {
+    height: $second-arrow-height;
+    left: calc(#{$center-clock-position} - 2px);
+    top: $center-clock-position - $second-arrow-height;
+    border: 2px solid #fff;
+    animation-duration: 5s;
+  }
+
+  &__minure-arrow {
+    height: $minute-arrow-height;
+    left: calc(#{$center-clock-position} - 4px);
+    top: $center-clock-position - $minute-arrow-height;
+    border: 4px solid #b2b75a;
+    animation-duration: 60s;
+  }
```

Comment:
you could create one base class, and one where you would override needed styles, like you did in task with two search bars